### DISCUSSION TITLE
分享经验里提到“抢票前，不要去提前抢还没有发售的票，会被b站封掉一段时间导致错过抢票的”，那么什么提前多久点开始抢票合适呢

### DISCUSSION THREAD

**Discussion by Kasakinozomizore** [mikumifa/biliTickerBuy#449] [提问专区]
估计抢票那天只能家里电脑开机挂着了，出门前点上？

**Comment by Kasakinozomizore** [comment]
![image](https://github.com/user-attachments/assets/7cceda44-a029-4fdd-b8a8-63d3c128f89d)

**Reply by Kasakinozomizore** [reply]
这个是不是算还没发售啊

**Reply by xhthyh** [reply]
同问 不想一人打车780回家

**Reply by Kasakinozomizore** [reply]
> 同问 不想一人打车780回家

![image](https://github.com/user-attachments/assets/b34ca2af-6492-4b99-966c-395c8c9538dc)
 在飞书的说明书里看到这个，我理解是不是比如12点开票，别设置成11:59开始抢的意思？
我设置完正点时间后，点了开始抢票，然后就是倒计时了，目前看着账号还没事儿？

**Reply by mikumifa** [reply]
> 我理解是不是比如12点开票，别设置成11:59开始抢的意思？ 我设置完正点时间后，点了开始抢票，然后就是倒计时了，目前看着账号还没事儿？

是这个意思

**Comment by misefeng** [comment]
请问gbc能抢到吗

**Reply by alphardex** [reply]
输了，抢不到一点
